osler/core/views.py

```diff
@@ -431,7 +432,12 @@ def all_patients(request, title='All Patients', active=False):
     """
     patient_list = core_models.Patient.objects.all()
     if active:
-        patient_list = patient_list.filter(needs_workup=True)
+        #if a patient has two open encounters, they will appear twice because of 
```

Comment:
suppose a patient does have two encounters open...is there any way for this to happen in the code? and would going into the admin panel be the only way to solve it?

Reply:
This should not happen... Since all patients are automatically inactive at creation and you create a new encounter by toggling active and when you toggle you deactivate it and that's the only way. Just in the future I guess if you have diff statuses idk. But yes going into the admin panel would be the only way to solve it. I figured that would be ok if it was a case that 1) would be difficult to make happen and 2) is bad and should be fixed if it did